python/paddle/v2/framework/tests/test_dynamic_recurrent_op.py

```diff
@@ -107,5 +111,58 @@ def test_forward(self):
         print 'pd_output', pd_output
 
 
+class RecurrentGradientOpTest(unittest.TestCase):
+    py = PyRNNStep()
+
+    def create_forward_op(self):
+        # create RNNOp
+        self.forward_op = DynamicRecurrentOp(
+            # inputs
+            inlinks=["x"],
+            boot_memories=["h_boot"],
+            step_net="stepnet",
+            # outputs
+            outlinks=["h@mem"],
```

Comment:
What is the convention to of the name `h@mem` and `h@pre`? What does the `@` here mean?

Reply:
Currently, the python API is not completed yet, so the high-level syntax as we described before cannot be used in this unit test. 
Both `h@mem` and `h@pre` are some underlying names of `memory` and `ex-memory`, the work to define these two names should be done by the python API in the future.